src/applications/gi/components/profile/Calculator.jsx

```diff
@@ -52,7 +52,7 @@ export class Calculator extends React.Component {
     const expanded = this.state.showEligibilityForm;
 
     return (
-      <div className="eligibility-details">
+      <div aria-live="off" className="eligibility-details">
```

Comment:
Shouldn't need to explicitly declare `aria-live="off"`. I don't see that listed on MDN ( https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions ) so let's remove it and try this out with the `<form>` wrapper.

Reply:
@ksantiagomorgan6 I'm okay leaving the `aria-live="off"` declaration. Long-term let's seek to remove the extraneous live regions and revisit this.

Reply:
Sounds good